default.json

```diff
@@ -3,5 +3,16 @@
   "extends": [
     "config:base",
     "helpers:pinGitHubActionDigests"
+  ],
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
+      "minimumReleaseAge": "3 days",
+      "schedule": ["every 2 weeks on Monday"]
```

Comment:
can we make this wednesday + minimum release age 4 days? too much happening on mondays

Reply:
here you go